relay/sources/rudder-pkg/lib/rudder-pkg/rudderPkgUtils.py

```diff
@@ -496,9 +500,14 @@ def list_plugin_name():
 
 p = Popen("rudder agent version", shell=True, stdout=PIPE)
 line = p.communicate()[0]
-m = re.match(r'Rudder agent (\d+\.\d+)\..*', line.decode('utf-8'))
+print(line.decode('utf-8'))
+m = re.match(r'Rudder agent (((\d+\.\d+)\.\d+)(\.((beta|rc)\d+))?)\..*?', line.decode('utf-8'))
```

Comment:
knowing it comes from the "rudder agent version" command which takes it from the package manager (hence different formatting depending on the platform), which got the information dynamically at build time by a Makefile edition with sed, coming from version.sh file makes me kind of sad.

Reply:
bit that's life. I can live with it.

Reply:
I can do otherwise if you want :) but don't know yet !! We could put it in rudder package at build time ...

Reply:
put a file containing the value in rudder agent at build time ?

Reply:
Would remove some steps in the 'rudder agent version' horror show

Reply:
As it is new code that only runs on the relay or root we could use the version from relayd (`/opt/rudder/bin/rudder-relayd --version`) which is more consistent and self-contained. I'm not sure if it's actually better though.